### DISCUSSION TITLE
Xmonad key table & startup app

### DISCUSSION THREAD

**Discussion by PlumpMath** [gvolpe/nix-config#132] [Q&A] [answered]
I have experience with xmonad, but not with haskell. (So I don't know the key by looking at the settings.)
It would be nice to have a shortcut table in the xmonad directory or on the root main screen.

Sorry to ask for more, but I wish it was as easy for disabling and opening a few startup apps.
For example messengers and file cloud

Thank you.

**Comment by gvolpe** [comment, marked as answer]
Super+F1 shows you the available key-bindings

![keybindings](https://i.imgur.com/WXethL5.png)
